src/components/genericListMaker.js

```diff
@@ -95,6 +99,15 @@ const genericListMaker = (componentName, FormComponent) => {
                       }}
                     />
                   ) : null}
+                  {editMode &&
```

Comment:
What is `editMode` exactly? :sweat_smile:

Reply:
I am not sure why it was added by previous developers in the first place. Since its value has been hardcoded to `true` right from the start, and forever remains true only! :sweat: